routes.js

```diff
@@ -29,6 +29,25 @@ function fastifySwagger (fastify, opts, next) {
     }
   })
 
+  // fastify.ready((err) => {
+  //   if (err) {
+  //     throw err
+  //   }
+  // })
+  const allSchemas = fastify.getSchemas()
```

Comment:
@Eomm, do you know if in v3 this method was reworked, so it will return all schemes, even those what were registered in child scopes with fastify.register?
Cause in fastify v2 I had to go through all child scopes in order to get all of them like https://github.com/SkeLLLa/fastify-oas/blob/master/lib/openapi/index.js#L5

Reply:
nope, it is encapsulated and tested here:
https://github.com/fastify/fastify/blob/master/test/schema-feature.test.js#L505

I agree that we could think a nicer solution 👍

Reply:
I think looping through children is quiet fine, unless you have schemes with same names. But in fastify-oas module there were no such issues, so it can be used.

Reply:
In think the onRegister hook could let us to avoid to use the symbol, I will give it a try

Reply:
One more idea - is to add addSchema hook to Fastify. That will allow plugins to get schemes when they are registered. And, for example, instead of resolving them we could put schema to swagger `definitions` and use `$ref` inside swagger route to that definition.